etc/opensc.conf.in

```diff
@@ -379,33 +379,11 @@ app default {
 		# driver = "piv";
 	# }
 
-	# Estonian ID card and Micardo driver sometimes only play together with T=0
+	# Micardo driver sometimes only play together with T=0
 	# In theory only the 'cold' ATR should be specified, as T=0 will
 	# be the preferred protocol once you boot it up with T=0, but be
 	# paranoid.
 	#
```

Comment:
I'm puzzled: Why do you still need this comment if the configuration below is deleted?

Reply:
Look below is D-Trust cards also Micardo cards

Reply:
thanks!